File: anul2/semestrul1/map/lab2/model/StackContainer.java
package model;

import java.util.ArrayList;
import java.util.List;

public class StackContainer extends AbstractContainer {
    public StackContainer() {
        super();
    }

    @Override
    public Task remove() {
        if (!tasks.isEmpty()) {
            return tasks.removeLast();
        }
        return null;
    }
}

File: Third Semester - Advanced Programming Methods/Lab10/model/StackContainer.cs
namespace lab10.model;

public class StackContainer:AbstractContainer
{
    public StackContainer() : base()
    {
        
    }
    
    public override Task Remove()
    {
        Task taskRemoved = tasks[^1];
        tasks.Remove(taskRemoved);
        return taskRemoved;
    }
}